src/unity/python/turicreate/__init__.py

```diff
@@ -25,7 +25,7 @@
 from .data_structures.sarray_builder import SArrayBuilder
 from .data_structures.sframe_builder import SFrameBuilder
 
-from turicreate.data_structures.sgraph import load_sgraph, load_graph
```

Comment:
We need to be really careful deleting these.  The main reason for the deprecation of load_graph is that it's required to load some saved models.  Before submitting this PR, we should test that all our load model tests pass.